subiquity/curtin.py

```diff
@@ -71,6 +71,11 @@
 # Reworked for flake8, but it does make it harder to read.
 POST_INSTALL_LIST = [
     ("late_commands:"),
+    ("    05_clear_subiquity: curtin in-target -- "
```

Comment:
I'm really not a fan of belt and braces like this. I'd much rather have something that did unsquashfs -ls installer.squashfs | grep subiquity && exit 1 or so in the image build.